terraform/modules/emqx-ee/main.tf

```diff
@@ -36,6 +36,29 @@ module "emqx" {
   module_depends_on       = var.module_depends_on
 }
 
+resource "null_resource" "wait_for_neg_creation" {
+  provisioner "local-exec" {
```

Comment:
Similar comments about the script above. Also, this 99% the same as the other script. The only differences (AFAICT), are object names, object types, and error messages. You can make all this dynamic.

Reply:
done